packages/core/src/hooks/useQueryParamState.ts

```diff
@@ -58,36 +58,35 @@ type SetQueryParams<T> = (params: T) => void;
 
 export function useQueryParamState<T>(
   stateName: string,
-  debounceTime: number = 100,
```

Comment:
Do we really need to break the API here?

Reply:
No, we don't but actually the parameter was introduced as a fix but isn't useful. I don't think that there are any other users right now. But I can reflect that in the changeset.

Reply:
@benjdlambert gave the PR an updated. I restored the parameter but marked it as deprecated.